Dockerfile

```diff
@@ -21,19 +21,26 @@ RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
 # Install PowerShell Az module
 ARG AZ_PWSH_VER=5.9.0
 RUN pwsh -noni -c "\$ProgressPreference='SilentlyContinue'; Install-Module Az -AllowClobber -RequiredVersion '${AZ_PWSH_VER}' -Repository PSGallery -Force -Scope AllUsers"
-ARG AZ_SYNAPSE_VER=0.11.0
+ARG AZ_SYNAPSE_VER=0.8.0
```

Comment:
Is this version intentionally downgraded?

Reply:
I noticed some assembly conflict issues, so I've erred on the side of caution and aligned the version with a key downstream consumer.

Reply:
However, Bicep has also been updated in the interim so I'm going to do another iteration of testing.